docs/API.md

```diff
@@ -32,7 +32,7 @@ source = <<-EOS
 EOS
 
 reporter = Reek::Report::TextReport.new
-examiner = Reek::Examiner.new(source)
+examiner = Reek::Examiner.new source
```

Comment:
Ok, veto complete :laughing: 

Now, of to serious internet bussiness: Can i get a merge pls?